### ISSUE TITLE
Conversion from / to Lists in Editor is problematic

### ISSUE THREAD

**Issue by Raisolution** [telerik/kendo-angular#2214]
Conversion is problematic in the following scenarios:

-  Direct converting headers into an ordered/unordered list
- Converting an ordered/unordered list to a paragraph/header
- Indentation of an ordered/unordered list

Issues are reproducible on the [following documentation page](https://www.telerik.com/kendo-angular-ui/components/editor/tools/).

**Comment by ZbigniewKrzystanek** [comment]
Hi, is it possible to say when this bug will be fixed?

**Comment by ZbigniewKrzystanek** [comment]
Hi,
when are you going to fix it?
Thanks

**Comment by ZbigniewKrzystanek** [comment]
Hi,
when are you going to fix it?
Thanks

**Comment by valchev** [comment]
@ZbigniewKrzystanek the issue will be fixed in the next official release which is scheduled for mid-September.

**Comment by ZbigniewKrzystanek** [comment]
> @ZbigniewKrzystanek the issue will be fixed in the next official release which is scheduled for mid-September or earlier in the dev builds.

Hi @valchev , I've checked 0.8.0 version of the editor and there is still no possibility to:

- Direct converting headers into an ordered/unordered list
- Converting an ordered/unordered list to a paragraph/header
- Indentation of an ordered/unordered list

Maybe I'm doing something wrong?

**Comment by valchev** [comment]
Hi @ZbigniewKrzystanek 

The aforementioned fixes are not available in the latest version, please accept my apology for the inconvenience caused. I understand that lists are a crucial functionality for the Editor users, so I will raise the priority of the issue to 'high' level.

**Comment by ZbigniewKrzystanek** [comment]
Hi @valchev , I've checked changes from package 0.11.0-dev.201911281341, so i have 2 things:
1. Indentation works only on elements in list after first, Is it possible to work also for a first element of a list?
2. There is no option to direct convert ordered/unordered list to a paragraph/header

Maybe I've missed something, how to do that, but kendoEditorCleanFormattingButton do not clean Format (e.g I have H1, Im using clean and I want to have plain text, a paragraph without heading ), the same is for ordered/unordered list( e.g I have ordered list which is selected, use clean and I want to have plain text, a paragraph without order or bullet points)?
